boards/arm/frdm_k64f/board.cmake

```diff
@@ -16,3 +16,4 @@ board_runner_args(pyocd "--target=k64f")
 include(${ZEPHYR_BASE}/boards/common/pyocd.board.cmake)
 include(${ZEPHYR_BASE}/boards/common/jlink.board.cmake)
 include(${ZEPHYR_BASE}/boards/common/openocd.board.cmake)
+include(${ZEPHYR_BASE}/boards/common/canopen.board.cmake)
```

Comment:
@mbolivar I would rather have included this in the sample `CMakeLists.txt` file, but that results in an error from `west flash --runner canopen`.

Reply:
>  I would rather have included this in the sample `CMakeLists.txt` file

I can see why.

I guess the feature request is to allow applications to define their own runners and configure arguments for them, right?

Reply:
Precisely.

Reply:
https://github.com/zephyrproject-rtos/zephyr/issues/23452 to track this.

I want to finish a few other things before looping back to new runner code, though.